src/app/data/forms/form-control.ts

```diff
@@ -0,0 +1,33 @@
+import { Control } from './control';
+
+export abstract class FormControl extends Control {
+  ccid: number = undefined;
+  readonly?: boolean = undefined;
+  autofill?: string = undefined;
```

Comment:
Macht autofill außerhalb eines Textfelds überhaupt Sinn? In der WollMux-Doku wird das nur mit textfield verwendet,

Reply:
autofill ist nicht eine Vorbelegung sondern berechnet abhängig von anderen Formularfeldern den Wert. Laut Doku kann es mit jedem Control vorkommen.